src/drivers/dw/dma.c

```diff
@@ -22,6 +22,7 @@
 
```

Comment:
@tlauda can you explain a little bit more the problem? If this is an urgent fix then I'm ok with it as it is. But other than that we really need commit message to explain the problem in detail.

Reply:
@dbaluta It's an ugly solution, but this fix is urgent to make more CI tests to pass. I have already almost done with the better one, but it's huge change and needs to wait for the rest of the major changes to be merged.